src/main/java/com/odysseusinc/arachne/executionengine/service/impl/AnalysisServiceImpl.java

```diff
@@ -117,6 +122,7 @@ public AnalysisRequestStatusDTO analyze(AnalysisSyncRequestDTO analysis, File an
         AnalysisRequestTypeDTO status = AnalysisRequestTypeDTO.NOT_RECOGNIZED;
         Future executionFuture = null;
         try {
+            File keyFile = Objects.equals(DBMSType.BIGQUERY, analysis.getDataSource().getType()) ? prepareBQAuth(analysis.getDataSource()) : null;
```

Comment:
Where is the `keyFile` variable used? Isn't this covered by `BigQueryAuthResolver`?

Reply:
agree
reference to key file was removed